### PR TITLE
CAP: send approved email + CAP: Publish alert button added + whitespace removal

### PR DESCRIPTION
**Pull request by biplovbhandari**
@flavour Please merge

### CODE REVIEW

File: controllers/cap.py [hunk 1]
```diff
@@ -1013,6 +1012,24 @@ def compose():
     redirect(URL(c="cap", f="alert"))
 
 # -----------------------------------------------------------------------------
+def alert_approve():
```
Comment (nursix) [inline controllers/cap.py]: Hmm - yesterday Fran said he wanted this to become an S3Method of cap_alert.

Now you merely renamed the controller - what am I missing here?

Comment (flavour) [inline controllers/cap.py]: Was exactly my comment earlier

Comment (biplovbhandari) [inline controllers/cap.py]: This was done as separate function as in others also as here:https://github.com/flavour/eden/blob/master/controllers/vulnerability.py#L1535
Besides, S3crud always outputs view https://github.com/flavour/eden/blob/master/modules/s3/s3crud.py#L790? This is just action? What am I missing here?

Comment (flavour) [inline controllers/cap.py]: > This was done as separate function as in others also as here: https://github.com/flavour/eden/blob/master/controllers/vulnerability.py#L1535

This is a custom function highly tied to the Vulnerability model, whereas I see nothing CAP-specific about the one you created. (Also the function was created bya  developer who wasn't aware of the S3Method-based approach)

Comment (flavour) [inline controllers/cap.py]: > S3crud always outputs view https://github.com/flavour/eden/blob/master/modules/s3/s3crud.py#L790? This is just action?

Typically yes, but I don't see that as a complete requirement.

**Pull request closed without merging**